src/main/java/frc/robot/subsystems/LiftSubsystem.java

```diff
@@ -83,6 +91,34 @@ public Command disengageBrake() {
         });
     }
 
+    public Command autoMoveLift(Heights target) {
+        return new Command("Auto Move Lift", this) {
+            @Override
+            protected void initialize() {
+                brake.set(Value.kReverse);
+                heightPID.reset();
+                heightPID.setSetpoint(target.value);
+                heightPID.enable();
+            }
+
+            @Override
+            protected void execute() {
+                motor.set(heightCorrection);
```

Comment:
We probably want to use the limit switches to ensure we don't accidentally drive into the ends.